src/app/mutation-page/mutation-page.component.ts

```diff
@@ -0,0 +1,54 @@
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { injectIsMutating, injectMutation } from '@ngneat/query';
+import { HttpClient } from '@angular/common/http';
+import { FormsModule } from '@angular/forms';
+
+interface Todo {
+  id: number;
+  title: string;
+  completed: boolean;
+}
+
+@Component({
+  selector: 'query-mutation-page',
+  standalone: true,
+  imports: [CommonModule, FormsModule],
+  templateUrl: './mutation-page.component.html',
+  styles: [],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MutationPageComponent {
+  private useIsMutating = injectIsMutating();
+  private useMutation = injectMutation();
+  private httpClient = inject(HttpClient);
+
+  public addTodoMutationsActive = this.useIsMutating({mutationKey: ['addTodo']}).toSignal();
```

Comment:
I think we can remove this and the mutationKey from addTodo  mutationFn. You'll not need it in 99% of the cases probably and I don't want people think that it's required

Reply:
Yeah makes sense, i just did it for the purpose of showing of the `isMutating` functionality, but this could also be done without any `mutationKey`. I'll remove it after work today.